docs/source/create_channel/create_channel_participation.md

```diff
@@ -0,0 +1,550 @@
+
+**Benefits of the new process:**
+
+ * **Increased privacy**: Because all ordering nodes used to be joined to the system channel, every ordering node in a network knew about the existence of every channel on that ordering service, even if the node itself wasn't a consenter on that channel and therefore wasn't ordering or storing its blocks. Now, an ordering node only knows about the channels it is joined to.
+ * **Scalability**: When there is a large number of ordering nodes on the system channel, starting the node can take a long time as it has to wait for each orderer to replicate the system channel ledger showing it has joined. Additionally, large numbers of nodes on the system channel increases the number of "alive" messages being sent between nodes. In large system channels with a large number of nodes, this can cause processing issues, even if no application channels have a large number of ordering nodes on it.
+ * **Flexibility**: Whereas ordering nodes could only be part of a single ordering service, orderers can now join or leave channels as needed, similar to how peers can join any channel its organization is a member of.
+ * **Operational benefits**:
+    * Easy to list the channels that the ordering node is a consenter on.
+    * Simple process to delete a channel and the ledger associated with that channel.
+    * Ordering nodes now detect that they have been added to a channel more quickly. Previously, this process could take several minutes, causing ordering node admins to restart their nodes so that the node could detect that it was joined more quickly.
+    * If the MSP of a peer organization, as listed in the system channel, needs to be changed, the peer organization no longer needs to coordinate with an orderer organization to change the MSP.
+
+**Note: the new mode of creating channels is incompatible with creating a channel using a system channel. If the system channel exists, the `Join` and `Remove` operations will not be supported. Similarly, they cannot be used with a Solo or Kafka ordering service.**
+
+Note: “Mixed mode” management, where the system channel is to create channels on some ordering nodes and the new process is used to create channels on other ordering nodes is not supported and highly discouraged. You must either transition to the new process or continue to use the system channel process. For information about removing the system channel from an existing ordering service as part of transitioning to the new process, check out [Remove the system channel](#remove-the-system-channel).
+
+In the process of creating the channel, this tutorial will take you through the following steps and concepts:
+
+- [Prerequisites](#prerequisites)
+- [Step one: Generate the genesis block of the channel](#step-one-generate-the-genesis-block-of-the-channel)
+- [Step two: Use the osnadmin CLI to create the channel](#step-two-use-the-osnadmin-cli-to-create-the-channel)
```

Comment:
I suppose this could devolve into a philosophical debate, but I would argue the channel is created when the genesis block is created.  The channel isn't very useful though until the orderer joins and begins to participate in consensus for that channel.

I'm also a bit confused why joining the first node is somehow a special step, while joining additional nodes is a separate step.  The channel will not be usable until enough nodes to form quorum have been joined.

Reply:
We decided to include 2 steps here because 
Step two is about  *consenters* and step three is about *followers*.
It felt like a logical way to break up all the content. 

I do see your point though. I changed it to 
Step two: Use the `osnadmin` CLI to add the first orderer to the channel
although in that step we join all 3 orderers so I'm thinking of simplifying it to 
Step two: Use the `osnadmin` CLI to join the orderers to the channel
But then 
Step three: Join additional ordering nodes
is confusing. I agree. I need to think on this a little more. I was trying to avoid using the terms consenter/ follower since is has not been introduced yet in the topic.